mirage/yaml/yaml_generator.py

```diff
@@ -379,6 +389,13 @@ def create_inputs(self):
             file_dict['primary_dither_num'] = np.int(primary_dither)
             subpix_dither = (tot_dith-1) % subpixtot
             file_dict['subpix_dither_num'] = subpix_dither + 1
+            file_dict['subarray_def_file'] = self.global_subarray_definition_files[self.info['Instrument'][i].lower()]
+            file_dict['readpatt_def_file'] = self.global_readout_pattern_files[self.info['Instrument'][i].lower()]
+            file_dict['crosstalk_file'] = self.global_crosstalk_files[self.info['Instrument'][i].lower()]
+            file_dict['filtpupilcombo_file'] = self.global_filtpupilcombo_files[self.info['Instrument'][i].lower()]
+            file_dict['flux_cal_file'] = self.global_flux_cal_files[self.info['Instrument'][i].lower()]
+            file_dict['psfpath'] = self.global_psfpath[self.info['Instrument'][i].lower()]
+
             fname = self.write_yaml(file_dict)
```

Comment:
Also further down, at line 432 in `yaml_generator`, it breaks for OTE-12 with the SUB96DHSPILA aperture. This should probably be fixed in the same PR that addresses #190, if you agree that should be separate.

Reply:
At least the observation list generation ran fine with OTE-12, so maybe that is solved?